Sources/_StringProcessing/Compiler.swift

```diff
@@ -101,6 +109,60 @@ class Compiler {
     }
   }
 
+  func compileLookaround(
+    _ kind: AST.Group.Kind,
+    _ child: AST
+  ) throws {
+    switch kind {
+    case .lookbehind, .negativeLookbehind:
+      throw unsupported("\(kind) assertions")
+
+    case .lookahead, .negativeLookahead:
+      break
+
+    default: fatalError("unreachable")
+    }
+
+    let positive = kind == .lookahead
+    assert(positive || kind == .negativeLookahead)
+    _ = """
```

Comment:
Can we use a comment instead? What's the benefit of using a string literal?

Reply:
Looks prettier in the editor (and sometimes better indentation than a multi-line comment), but I can use comments